src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+    private static final String REPORT_HEADER_SUPPLY = "supply";
+    private static final String REPORT_HEADER_BUY = "buy";
+    private static final String REPORT_HEADER_RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Please split public method into 3 parts (reading from file, preparing report, writing to file) and extract each part to separate private method.
Then call these 3 methods from public one to make it short and clear for understanding